docs/modules/ROOT/pages/how-tos/multi-instance.adoc

```diff
@@ -0,0 +1,60 @@
+= Deploy multiple instances
+
+This guide provides an example how to deploy multiple instances of this component.
+
+== Requirements
+
+
+. Prepare catalog by configuring 2 instances
++
+[source,yaml]
+----
+applications:
+  - exoscale-metrics-collector as collector-ch-gva-2 # alias collector-ch-gva-2
+  - exoscale-metrics-collector as collector-rma-0 # alias collector-rma-0
+parameters:
+  appuio_cloud_reporting:
+    namespace: 'appuio-cloud-reporting'
+    database:
+      name: 'reporting'
+      host: 'reporting-db.appuio-reporting.svc'
+      parameters: 'sslmode=disable'
+      password: 'passw0rd'
+      port: 5432
+  collector_ch_gva_2:
+    namespace: 'appuio-cloud-reporting'
+    secrets:
+      cronjob:
+        stringData:
+          EXOSCALE_API_KEY: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/exoscale-key}"
+          EXOSCALE_API_SECRET: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/exoscale-secret}"
+          K8S_SERVER_URL: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/cluster-server}"
+          K8S_TOKEN: "?{vaultkv:${cluster:tenant}/${cluster:name}/exoscale-metrics-collector/${_instance}/cluster-token}"
```

Comment:
Ah I understood what you meant, the previous version was the same I just added the k8s env variables. Also I don't see here any problems here having detailed explanation. Do we really need to lose time at this level of nitpicking?

Reply:
This specific configuration will be used only once and will most definitely be done by us. SO in the end whoever will use it will have to change the configuration.